javascript/javascript3/week3/screenshot/screenshot.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="screenshot.css">
+    <title>Screen shot</title>
+</head>
+
+<body>
+    <main>
+        <h1 class="heading-1">Get a Screenshot</h1>
+        <form id="create-new-screenshot">
+            <fieldset>
+                <legend>Screen-Shot-Generate And Saved</legend>
+                <div class="form-field">
+                    <label class="label" for="add-screenshot">Screen-Shot-url*</label>
+                    <input type="text" class="input" id="add-screenshot" />
+                    <button class="button" id="save">Save</button>
+                </div>
+            </fieldset>
+        </form>
+        <div class="screenshot-input">
+            <h3>Show-ScreenShot-Url</h3>
```

Comment:
HTML headings are titles or subtitles that you want to display on a webpage.
```suggestion
            <h3>ScreenShot's Url</h3>
```

Reply:
this should also be hidden before you have screenshot